docs/documentation/cl_options/index.rst

```diff
@@ -0,0 +1,40 @@
+Command Line Arguments
+======================
+
+What starting vimiv form the command line you have the ability to pass a number of
+different argument to vimiv. The general calling structure is depicted below:
+
+.. include:: synopsis.rstsrc
+
+Command Line Arguments
+----------------------
+
+The following is an exhaustive list of all available arguments:
+
+.. include:: options.rstsrc
+
+Examples
+--------
+
+In the following we present a few use cases of command line arguments.
+
+* Start in thumbnail view instead of image view
+    ``vimiv * --command "enter thumbnail" --command "enter library"``
+
+* Start in read-only mode. This prevents accidental modification (renaming, moving, editing etc.) of any images
+    ``vimiv --set read_only true``
+
+* Change WM_CLASS_INSTANCE to identify a vimiv instance
+    ``vimiv --qt-args '--name myVimivInstance'``
+
+* Print the last selected image to STDOUT when quitting
+    ``vimiv --output "%"``
+
+* Use vimiv as *Rofi for Images* to make a selection from candidate image
+    ``mySel=$(echo $myCand | vimiv --input --output "%m" --command "enter thumbnail")``
+
+* Print debug logs of your amazing plugin you are writing and of the `api._mark` module which does not behave as you are expecting
+    ``vimiv --debug myAmazingPlugin api._mark``
+
+* Collect data to report this weird bug you are experiencing
```

Comment:
Curious what the use-case of this is compared to just using `~/.local/share/vimiv/vimiv.log` with `vimiv --log-level debug`

Reply:
Ok well, your right. I have removed it in e6a562414448933601ee2b9b8f0000c9fe266fc2